ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/FloorKudf.java

```diff
@@ -19,12 +19,18 @@
 import io.confluent.ksql.function.udf.Kudf;
 
 public class FloorKudf implements Kudf {
+  public static final String NAME = "FLOOR";
 
   @Override
   public Object evaluate(final Object... args) {
-    if (args.length != 1) {
-      throw new KsqlFunctionException("Floor udf should have one input argument.");
+    if (args.length != 1  && args.length != 2) {
+      throw new KsqlFunctionException("Floor udf should have one or two input arguments.");
     }
-    return Math.floor((Double) args[0]);
+    final Double number = (Double) args[0];
+    if (args.length == 1) {
+      return (long) Math.floor(number);
```

Comment:
will result in NPE if `number` is null.   This would result in a log of errors being logged unnecessarily. We should just return `null` if number is `null`.

A unit test would be good to test this case, as QTT tests can't catch such an issue.  

We should also check the second arg for `null`.  If provided, but `null`... then what we do is debatable. Maybe just ignore it, i.e. a `null` second arg just the same as having only 1 arg.